<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesWindow</class>
 <widget class="QMainWindow" name="PropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertiesWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>362</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="dataset_label">
       <property name="text">
        <string>DATASET #XXX</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QCheckBox" name="tl_enabled">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QCheckBox" name="tr_enabled">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="tl_label">
         <property name="text">
          <string>Top Left</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QCheckBox" name="three_d_enabled">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QCheckBox" name="bl_enabled">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="tl_combo_b">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="bl_combo_a">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="tr_label">
         <property name="text">
          <string>Top Right</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="bl_label">
         <property name="text">
          <string>Bottom Left</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="three_d_label">
         <property name="text">
          <string>3D Plot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="tr_combo_b">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="tr_combo_a">
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="bl_combo_b">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="tl_combo_a">
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
